### PR TITLE
Added steps to embed the badges

### LINKED ISSUE #129: New section on how to embed the badges on other profiles
#### Current Behavior
Right now https://badges.layer5.io/ needs some content and one of sections that can be added is embedding the badges on other social profiles

#### Desired Behavior
A section on how to embed the badges in your other social profiles like GitHub profile and LinkedIn profile.

Reference:
Meshery Cloud -> Profile -> Click on the badge-> Copy embedded code

tempbadge

---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by gopivaibhav**
**Description**

This PR fixes #129 

**Notes for Reviewers**

- Centered the stick figure graphic
- Added steps on how to embed the badge
- Added Karan's GitHub profile as another example 

image

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: site/src/pages/index.js [hunk 1]
```diff
@@ -53,8 +53,18 @@ const App = () => {
               <BadgeGrid  />
             </section>
             <div className="banner-logos">
-              <h4> Take pride and embed your Layer5 badges in your GitHub profile</h4>
-              <img src={githubBadges} alt="Recognition Program Banner"/>
+              <h3> Take pride and embed your Layer5 badges in your GitHub profile</h3>
```
Comment (leecalcote) [inline site/src/pages/index.js:56]:
```suggestion
              <h3> Wear your badge pride proudly. Share and embed your Layer5 badges in your GitHub profile</h3>
```

File: site/src/pages/index.js [hunk 2]
```diff
@@ -53,8 +53,18 @@ const App = () => {
...
-              <h4> Take pride and embed your Layer5 badges in your GitHub profile</h4>
-              <img src={githubBadges} alt="Recognition Program Banner"/>
+              <h3> Wear your badge pride proudly. Embed your Layer5 badges in your GitHub profile</h3>
+              <h4>Follow these steps to display your badge:</h4>
+              <ul>
+                <li>Visit your Layer5 Cloud Profile to see the badges</li>
+                <li>Click on the badge, which you wanted to display</li>
+                <li>Copy the markdown code by clicking the Embed Code icon</li>
+                <li>Paste the code in your GitHub profile README.</li>
+              </ul>
```
Comment (leecalcote) [inline site/src/pages/index.js:63]:
```suggestion
              <ol>
                <li>Visit your Layer5 Cloud Profile to see the badges</li>
                <li>Click on the badge, which you wanted to display</li>
                <li>Copy the markdown code by clicking the Embed Code icon</li>
                <li>Paste the code in your GitHub profile README.</li>
              </ol>
```

Comment (leecalcote) [reply site/src/pages/index.js:63]: Is this an improvement?

File: site/src/pages/index.js (hunk 1 above)
Comment (gopivaibhav) [reply site/src/pages/index.js]: I have stolen this phrase from the Layer5 cloud profile badge card actually :P

**Pull request merged**